src/pkg/packager/create.go

```diff
@@ -56,6 +57,31 @@ func (p *Packager) Create(baseDir string) error {
 		return err
 	}
 
+	indexesToInsertFluxComponent := []int{}
+
+	// do bigbang image/repo discovery
+	for i, c := range p.cfg.Pkg.Components {
```

Comment:
This should but pulled into it's own private function, but also feels very gross to mutate like that, would like to think through the Flux usage here a bit more. We intentionally moved hardcoded things like this out of the binary and into the init package previously. It feels like maybe we just need a Big Bang init package instead for this.